KMNavigationBarTransition/UINavigationController+KMNavigationBarTransition.m

```diff
@@ -49,6 +49,10 @@ + (void)load {
     });
 }
 
+- (UIColor *)km_containerViewBackgroundColor {
```

Comment:
I think it is better to put it in `UIViewController+KMNavigationBarTransition.m`.